coreneuron/permute/cellorder.cpp

```diff
@@ -656,28 +650,23 @@ void solve_interleaved1(int ith) {
     int* firstnode = ii.firstnode;
     int* lastnode = ii.lastnode;
     int* cellsize = ii.cellsize;
-#if _OPENACC
-    int stream_id = nt->stream_id;
-#endif
 
-#ifdef _OPENACC
-    // clang-format off
-
-    #pragma acc parallel loop present(              \
-        nt[0:1], stride[0:nstride],                 \
-        firstnode[0:ncell], lastnode[0:ncell],      \
-        cellsize[0:ncell]) if (nt->compute_gpu)     \
-        async(stream_id)
-// clang-format on
-#endif
+    // OL211123: can we preserve the error checking behaviour of OpenACC's
+    // present clause with OpenMP? It is a bug if these data are not present,
+    // so diagnostics are helpful...
```

Comment:
// nitpick/note: this we could remove from here, we will post question to mentors.

Reply:
Decided to leave this in place to help ensure we reach a conclusion.